components/nft-item/index.tsx

```diff
@@ -81,15 +93,15 @@ const NFTItem = forwardRef<HTMLDivElement, NFTItemProps>(
     }
 
     let showAmount = "";
-    if(amount!=undefined){
-      const BIamount = utils.parseUnits(String(amount),"wei");
-      if(BIamount.lt(utils.parseEther("1.0"))){
-        showAmount="<0.01";
-      }else{
-        showAmount=utils.formatEther(BIamount.div(100));
+    if (amount != undefined) {
+      const BIamount = utils.parseUnits(String(amount), "wei");
+      if (BIamount.lt(utils.parseEther("1.0"))) {
+        showAmount = "<0.01";
+      } else {
+        showAmount = utils.formatEther(BIamount.div(100));
       }
-      
-    }else{
+
+    } else {
       showAmount = "";
```

Comment:
Looks like this `else` block  is removable

Reply:
Great insight. I'm on it.